shared/uavobjectdefinition/stabilizationsettings.xml

```diff
@@ -30,7 +30,7 @@
 	<field name="VbarPiroComp" units="" type="enum" elements="1" options="FALSE,TRUE" defaultvalue="FALSE"/>
 	<field name="VbarMaxAngle" units="deg" type="uint8" elements="1" defaultvalue="10"/>
 
-	<field name="GyroTau" units="" type="float" elements="1" defaultvalue="0.005"/>
+	<field name="GyroCutoff" units="Hz" type="float" elements="1" defaultvalue="40.6"/>

```

Comment:
In cleanflight with the same filter 60Hz seems to be the favorite.
But as i have not much flight experience, and my quad can't fly at the moment, i wanted to start with a conservative default value.

But when there are recommendations for better default values, i like to change this.